src/useCases/redirectUserTo.ts

```diff
@@ -0,0 +1,16 @@
+import { GetServerSidePropsResult, Redirect } from "next";
+
+export interface IRedirectUserTo {
+  execute: (url: string) => GetServerSidePropsResult<Redirect>;
```

Comment:
I'm not sure what should be in those angled brackets as in `GetServerSidePropsResult<HERE>` so this might be very wrong, I'm going to have a google to try and understand what that even is.